squarelet/oidc/utils.py

```diff
@@ -19,3 +19,15 @@ def send_cache_invalidations(model, uuids):
         logger.info("Sending cache invalidations for: %s %s", model, uuids)
         for client_profile in ClientProfile.objects.exclude(webhook_url=""):
             tasks.send_cache_invalidation.delay(client_profile.pk, model, uuids)
+
+
+def oidc_login_hook(request, user, client):
+    """Log which client users login to"""
+    # take an arbitrary non-individual organization, since most users will have one org
+    organization = user.organizations.filter(individual=False).first()
+    plan = None if organization is None else organization.plan
```

Comment:
Is it worth trying to figure out if a user belongs to _any_ org with a premium plan here?

Reply:
Possibly?  I didn't devote too much time thinking about it as I assume the vast majority of users are only in one org.  Depending on how accurate we need to be, it might be worth doing something smarter